pkg/osutil/interrupt_unix.go

```diff
@@ -69,6 +69,11 @@ func HandleInterrupts() {
 			os.Exit(0)
 		}
 		syscall.Kill(pid, sig.(syscall.Signal))
+
+		// unlock exit lock since the go runtime doesn't reinstall
```

Comment:
why do we want to unlock after we call syscall.Kill with the original signal? shouldnt etcd already be killed? why do we want it to exit by itself?